steps/applivery-deploy-ios/0.1/step.yml

```diff
@@ -0,0 +1,119 @@
+title: Applivery.com iOS Deploy
+summary: Deploy your awesome iOS Application to Applivery.com
+description: |-
+  Deploy an iOS application to [Applivery](http://www.applivery.com),
+    add notes and notify testers.
```

Comment:
Drop the additional indentation. The `description` property is handled as a Markdown text, additional indentation might change the type of the text (e.g. to `code` tag).